### PR TITLE
New ML sections

### PR DESCRIPTION
**Pull request by carolinepacheco**
Added Training Custom AI Workflows - Pytorch library and Prediction Custom AI Workflows - Pytorch library sections.

### CODE REVIEW

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 1]
```diff
@@ -1848,24 +1848,34 @@ The following workflow presents an example of a workflow built using pre-built A
 
 *Custom_Sentiment_Analysis:* is a mashup that searches for news related to a given search term using Azure Bing News API then performs a sentiment analysis using a custom deep learning based pretrained model.
 
-=== Custom AI Workflows - Pytorch library
-
-The following workflows present examples of workflows built using custom AI tasks available on the `deep-learning` bucket. Such tasks enable you to train and test your own AI models by a simple drag and drop of custom AI task.
+=== Training Custom AI Workflows - Pytorch library
+The following workflows present examples of workflows built using custom AI tasks available on the `deep-learning` bucket. Such tasks enable you to train your own AI models by a simple drag and drop of custom AI task.
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc:1852]: of a custom AI task or of the custom AI task

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 2]
```diff
@@ -1848,24 +1848,34 @@ The following workflow presents an example of a workflow built using pre-built A
...
+
+*Train_Image_Object_Detection:* trains objects using YOLOv3 model on COCO dataset proposed by Microsoft Research. 
 
 *Deep_Model_Explainability:* explains a VGG16 model using GradientExplainer.
 
-*Image_Object_Detection:* detects objects using a pre-trained YOLO v3 model on COCO dataset proposed by Microsoft Research. The pre-trained model is available on this link.
+
+=== Prediction Custom AI Workflows - Pytorch library
+
+The following workflows present examples of workflows built using custom AI tasks available on the `deep-learning` bucket. Such tasks enable you to test your own AI models by a simple drag and drop of custom AI task.
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc:1870]: same

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 3]
```diff
@@ -1848,24 +1848,34 @@ The following workflow presents an example of a workflow built using pre-built A
...
+
+*Image_Segmentation:* predicts a segmentation model using SegNet network on http://www.robots.ox.ac.uk/~vgg/data/pets/[Oxford-IIIT Pet Dataset^]. The pre-trained image segmentation model which is available on this https://s3.eu-west-2.amazonaws.com/activeeon-public/models/model_segnet.zip[link^]
+
+*Fake_Celebrity_Faces_Generation:* generates a wild diversity of fake faces using a GAN model that was trained based on thousands of real celebrity photos. The pre-trained GAN model is available on this https://s3.eu-west-2.amazonaws.com/activeeon-public/models/Epoch+018.pt[link^]
+
+*Image_Segmentation:* predicts a segmentation model using SegNet network on http://www.robots.ox.ac.uk/~vgg/data/pets/[Oxford-IIIT Pet Dataset^]. The pre-trained image segmentation model which is available on this https://s3.eu-west-2.amazonaws.com/activeeon-public/models/model_segnet.zip[link^]
+
+*Image_Object_Detection:* detects objects using a pre-trained YOLOv3 model on COCO dataset proposed by Microsoft Research. The pre-trained model is available on this link.
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc:1878]: `on this link` which link? 
`COCO dataset proposed by Microsoft Research` do you have a reference to add here?

Comment (carolinepacheco) [reply src/docs/MLOS/MLOSUserGuide.adoc:1878]: Ok

**Pull request merged**